lib/kernel/src/application_master.erl

```diff
@@ -75,7 +75,7 @@ call(AppMaster, Req) ->
     AppMaster ! {Req, Tag, self()},
     receive 
 	{'DOWN', Ref, process, _, _Info} ->
-	    ok;
+	    error;
```

Comment:
The master process already returns `error` is the child
process is dead, so I choose to align it.

It should not affect anything, because all callers seem
to simply assume a tuple is returned. We only care about
this in case there is a race condition and then master
process terminates after we retrieve it.